contracts/implementations/RMRKNestableExternalEquipImpl.sol

```diff
@@ -73,6 +93,12 @@ contract RMRKNestableExternalEquipImpl is
         }
     }
 
+    /**
+     * @notice A hook to be called prior to minting tokens.
```

Comment:
Not really meant to be a hook, it's just to avoid code duplication on `mint` + `nestMint`.